packages/taquito/src/wallet/wallet.ts

```diff
@@ -257,6 +259,25 @@ export class Wallet {
     });
   }
 
+  /**
+   *
+   * @description failing_noop operation that is guaranteed to fail.
+   *
+   * @returns Signature for a failing_noop
+   *
+   * @param params operation parameter
+   */
+  async signFailingNoop(params: WalletFailingNoopParams) {
+    const failingOperation = await this.context.prepare.failingNoop(params);
+    const forgeable = this.context.prepare.toForge(failingOperation);
+    const protocolHash =
+      (this.context.proto as unknown as ProtocolsHash) ??
+      (await this.context.readProvider.getNextProtocol('head'));
+    const forger = new LocalForger(protocolHash);
```

Comment:
might not want to fixated to localforger when forge should use the one in the context can be local, remote or composite forger. the one in the context will be what user set up in TezosToolkit

Reply:
this.context.forger.forge(forgeable)